tests/test_base_api_client.py

```diff
@@ -53,7 +53,8 @@ def test_http_error_raises_api_error(self, base_client, rmock):
         with pytest.raises(HTTPError) as e:
             base_client._request("GET", '/')
 
-        assert e.value.message == REQUEST_ERROR_MESSAGE
+        assert e.value.message == "500 Server Error: None for url: http://baseurl/\n" \
+                                  "HTTPError('500 Server Error: None for url: http://baseurl/',)"
```

Comment:
Perhaps this isn't under your control here, but `None for url:...` isn't the most enlightening message to get.

Reply:
Yeah, this I don't think is under my control. It is at least a more descriptive error than what we currently get though ('Request failed'), which is the root goal here.